WordPress/src/main/java/org/wordpress/android/widgets/PostListButtonType.kt

```diff
@@ -24,7 +24,7 @@ enum class PostListButtonType constructor(
     BUTTON_BACK(10, R.string.button_back, R.drawable.ic_chevron_left_white_24dp, R.color.neutral_500),
     BUTTON_SUBMIT(11, R.string.submit_for_review, R.drawable.ic_reader_white_24dp, R.color.neutral_500),
     BUTTON_RETRY(12, R.string.button_retry, R.drawable.ic_refresh_white_24dp, R.color.error),
-    BUTTON_RESTORE(13, R.string.button_restore, R.drawable.ic_refresh_white_24dp, R.color.neutral_500);
+    BUTTON_MOVE_TO_DRAFT(13, R.string.button_move_to_draft, R.drawable.ic_refresh_white_24dp, R.color.neutral_500);
```

Comment:
Just noting here that the merge conflict was for this line in case you want to re-review it @malinajirka.